metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -314,6 +314,21 @@ def emit_card_mces(self) -> Iterable[MetadataWorkUnit]:
             )
             return None
 
+    def emit_card_id(self, card_id: int) -> Source:
+        card_url = f"{self.config.connect_uri}/api/card/{card_id}"
+        try:
+            card_response = self.session.get(card_url)
+            card_response.raise_for_status()
+            card_details = card_response.json()
+
+            return card_details
+        except HTTPError as http_error:
+            self.report.report_failure(
```

Comment:
If you report failure here, that will fail the whole ingestion in the end.
Shouldn't this be a warning?

Reply:
True, it should not fail the whole ingestion. I will change to warning

Reply:
solved: https://github.com/datahub-project/datahub/pull/6729/commits/7811f3a2cc328f98c7c14a960ef3b86615e95c8a